tensorflow_datasets/scripts/test_generate_visualization.py

```diff
@@ -0,0 +1,20 @@
+import os
+import tensorflow_datasets as tfds
+from tensorflow_datasets.scripts import generate_visualization
+from tensorflow_datasets.scripts import document_datasets
+from tensorflow_datasets.scripts import document_dataset_with_example
+
+DATASET_TO_TESTS = ['beans']  # Datasets you want to test the script on.
+
+def main():
+    for ds_name in DATASET_TO_TESTS:
```

Comment:
Could you format your code using https://www.tensorflow.org/datasets/add_dataset#5_check_your_code_style ?

Reply:
I didn't realize I pushed that file,  just formatted it